src/features/rewards/walletWrapper/index.tsx

```diff
@@ -348,7 +350,7 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
 
     return (
       <StyledNotificationMessage>
-        <StyledTypeText>{typeText}</StyledTypeText> <StyledPipe>|</StyledPipe>
+        <StyledTypeText onClick={textAction}>{typeText}</StyledTypeText> <StyledPipe>|</StyledPipe>
```

Comment:
Please move action to the button. So that we would have instead of OK, we would say Backup Now

Reply:
@NejcZdovc Done

<img width="400" alt="screen shot 2019-02-12 at 7 45 51 am" src="https://user-images.githubusercontent.com/8732757/52644495-2e6a3900-2e9c-11e9-91a3-b52e8e41ece9.png">

Reply:
perfect, thx